From 95e42e9944f70640ec4d7a2ea4973401700bcd6b Mon Sep 17 00:00:00 2001
From: Mike Zeller <mike.zeller@joyent.com>
Date: Tue, 10 Oct 2017 15:05:06 -0700
Subject: [PATCH] NAPI-433 listNetworkIPs schema should accept owner_uuid (Doc
 typo) Reviewed by: Trent Mick <trentm@gmail.com> Approved by: Trent Mick
 <trentm@gmail.com>

---
 docs/index.md | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/docs/index.md b/docs/index.md
index b7d7974..c2d9318 100644
--- a/docs/index.md
+++ b/docs/index.md
@@ -687,7 +687,7 @@ List VLANs owned by a user.
 
 Create a new fabric VLAN.
 
-## Inputs
+### Inputs
 
 | Field              | Type             | Description                                                                            |
 | ------------------ | ---------------- | -------------------------------------------------------------------------------------- |
-- 
2.21.0

